README.md

```diff
@@ -1,48 +1,32 @@
 
-- [Table of Contents](#table-of-contents)
-  - [Introduction](#introduction)
-  - [Automation Host Prerequisites](#automation-host-prerequisites)
-  - [PowerVS Prerequisites](#powervs-prerequisites)
-  - [OCP Install](#ocp-install)
-  - [Contributing](#contributing)
+# Welcome
 
+This repository contains Terraform templates, configurations and automation which leverages the [helpernode ansible playbook](https://github.com/RedHatOfficial/ocp4-helpernode) to assist on deploying [OpenShift Container Platform (OCP) 4.x](https://www.openshift.com/products/container-platform) on [IBM® Power Systems™ Virtual Server on IBM Cloud](https://www.ibm.com/cloud/power-virtual-server).
 
-## Introduction
+# Contributing
 
-This repo contains Terraform templates to help deployment of OpenShift Container Platform (OCP) 4.x on [IBM® Power Systems™ Virtual Server on IBM Cloud](https://www.ibm.com/cloud/power-virtual-server).
+Take a look at [contributing](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/CONTRIBUTING.md) to learn how to help make this a better project.
 
-This project leverages the helpernode [ansible playbook](https://github.com/RedHatOfficial/ocp4-helpernode) internally for OCP deployment on IBM Power Systems Virtual Servers (PowerVS).
+:heavy_exclamation_mark: for any bugs, enhancement, requests, fixes, etc. [open a new GitHub issue](https://github.com/ocp-power-automation/ocp4-upi-powervs/issues/new) and ensure it is not duplicated.
 
-:heavy_exclamation_mark: *For bugs/enhancement requests etc. please open a GitHub issue*
+:heavy_exclamation_mark: [checkout the latest stable version](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/release-4.5).
 
-For general PowerVS usage instructions please refer to the following links:
-- https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started
-- https://www.youtube.com/watch?v=RywSfXT_LLs
-- https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv
+# PowerVS Preparation
 
+You need to handle an initial preparation in your PowerVS instance to successfuly execute this automation. Follow this [guide](docs/ocp_prereqs_powervs.md) to get it done. To get started with PowerVS, [click here](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started). You can also learn more about PowerVS by watching some YouTube videos by clicking [here](https://www.youtube.com/watch?v=RywSfXT_LLs) and [here](https://www.youtube.com/playlist?list=PLVrJaTKVPbKM_9HU8fm4QsklgzLGUwFpv).
 
-:heavy_exclamation_mark: **Ensure that you're working from the desired release-specific version of this `README` as you deploy by first selecting the appropriate branch from the drop-down above. At the time of this writing, the latest release is 4.5: [README](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/release-4.5). The `README` document located in the `master` branch changes frequently and should not be considered final.**
```

Comment:
Please do not delete this text for now. This is specifically added for first time users to try stable release.